renovate.json

```
@@ -1,6 +1,18 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:recommended"
-  ]
-}
+    ":dependencyDashboard",
+    ":semanticPrefixFixDepsChoreOthers",
+    ":gitSignOff",
+    ":ignoreModulesAndTests",
+    ":prConcurrentLimit10",
+    "workarounds:typesNodeVersioning"
+  ],
+  "lockFileMaintenance": {
+    "extends": [
+      "schedule:monthly"
+    ],
+    "enabled": true,
+    "automerge": true
+  }
+}
```

I wasn't able to find the reason why we did not use the `base` config for the Postgresql repo.

Ideally, we'd share the common config so that Renovate behaves consistently across Repos:

```suggestion
    "github>loopbackio/cicd//shared-configs/renovate/base"
}
```

The above would cover majority of the Postgresql config, except below:

We should upstream this config preset to `base`:

- `workarounds:typesNodeVersioning`

This however, should not be upstreamed:

- `:semanticPrefixFixDepsChoreOthers`
  This would change deps updates
into `fix:`. We use `chore:` for deps updates.